docs/command-reference.md

```diff
@@ -388,7 +388,7 @@ IPFS flags:
 Unimplemented `docker run` flags:
     `--attach`, `--blkio-weight-device`, `--cpu-rt-*`, `--device-*`,
     `--disable-content-trust`, `--domainname`, `--expose`, `--health-*`, `--isolation`, `--no-healthcheck`,
-    `--link*`, `--publish-all`, `--sig-proxy`, `--storage-opt`,
+    `--link*`, `--publish-all`, `--storage-opt`,
```

Comment:
Could you add the usage of sig-proxy?

Also, please squash the commits

Reply:
👍🏾 
Added it